README.rst

```diff
@@ -144,6 +99,52 @@ Matrix clients without needing to run Synapse with root privileges.
 For information on configuring one, see `the reverse proxy docs
 <https://matrix-org.github.io/synapse/latest/reverse_proxy.html>`_.
 
+Testing a new installation
+==========================
+
+The easiest way to try out your new Synapse installation is by connecting to it
+from a web client.
+
+Unless you are running a test instance of Synapse on your local machine, in
+general, you will need to enable TLS support before you can successfully
+connect from a client: see
+`TLS certificates <https://matrix-org.github.io/synapse/latest/setup/installation.html#tls-certificates>`_.
+
+An easy way to get started is to login or register via Element at
+https://app.element.io/#/login or https://app.element.io/#/register respectively.
+You will need to change the server you are logging into from ``matrix.org``
+and instead specify a Homeserver URL of ``https://<server_name>:8448``
+(or just ``https://<server_name>`` if you are using a reverse proxy).
+If you prefer to use another client, refer to our
+`client breakdown <https://matrix.org/docs/projects/clients-matrix>`_.
+
+If all goes well you should at least be able to log in, create a room, and
+start sending messages.
+
+.. _`client-user-reg`:
+
+Registering a new user from a client
+------------------------------------
+
+By default, registration of new users via Matrix clients is disabled. To enable
+it, specify ``enable_registration: true`` in ``homeserver.yaml``. (It is then
```

Comment:
To enable registration you now also have to set `enable_registration_without_verification` to be true unless you have email/captcha/whatever verification enabled, but this should mention that doing so will get their new homeserver inundated with spam.

Reply:
I've written something for this in 77f6b8894. Please take a look!

Reply:
Looks great!